newm-server/src/main/kotlin/io/newm/server/database/migration/V59__MarketplaceUpdates.kt

```diff
@@ -36,7 +35,7 @@ class V59__MarketplaceUpdates : BaseJavaMigration() {
                         total_bundle_quantity bigint NOT NULL
                     )
                     """.trimIndent(),
-                    "INSERT INTO config VALUES ('$CONFIG_KEY_MARKETPLACE_MONITORING_ENABLED','true') ON CONFLICT(id) DO NOTHING",
+                    "INSERT INTO config VALUES ('marketplace.monitoringEnabled','true') ON CONFLICT(id) DO NOTHING",
```

Comment:
shouldn't you use this var instead? `CONFIG_KEY_MARKETPLACE_MONITORING_MULTI_MODE_ENABLED`

Reply:
`monitorEnabke` is correct. It's the old var here.